config/jobs/kubernetes-sigs/aws-ebs-csi-driver/aws-ebs-csi-driver-presubmits.yaml

```diff
@@ -2,10 +2,10 @@ presubmits:
   kubernetes-sigs/aws-ebs-csi-driver:
   - name: pull-aws-ebs-csi-driver-test-e2e-external-eks-windows
     cluster: eks-prow-build-cluster
-    always_run: true
     decorate: true
     skip_branches:
     - gh-pages
+    skip_if_only_changed: "(^docs/)|(^examples/)|(.md$)|(OWNERS$)|(^.github/)"
```

Comment:
Drop the parenthesis, otherwise lgtm

Reply:
Looks like it fits in line with other projects like k-sigs/kind